src/useSignIn.tsx

```diff
@@ -60,7 +60,7 @@ function useSignIn(
       fieldError,
       formState,
       isFormValid,
-      signingIn,
+      signingIn: !!signingIn,
```

Comment:
isn't it better to ensure croods always returns a boolean instead of doing it on every usage?

Reply:
The error appeared on croods-auth, I followed the easy way :worried:. What would be the option?